# Face visual features
float64[] features
